store/src/java-test/com/zimbra/cs/service/formatter/MobileConfigFormatterTest.java

```diff
@@ -0,0 +1,273 @@
+/*
+ * ***** BEGIN LICENSE BLOCK *****
+ * Zimbra Collaboration Suite Server
+ * Copyright (C) 2011, 2012, 2013, 2014, 2016 Synacor, Inc.
```

Comment:
It's new class, copyright should just have 2018 year.

Reply:
done